releasenotes/notes/31380.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: security
+issue:
+- 31380
+releaseNotes:
+- |
+  **Added** the support of configuring the cipher suites for inbound TLS connections.
```

Comment:
Why only inbound and not outbound? W kind of inbound connections are we talking about here? Is it only for sidecar-to-sidecar or also at gateways?

Reply:
This is only for inbound sidecar-to-sidecar. The information we get from users is that they like to configure the TLS cipher suites on the server side (i.e., inbound).